admin-ui/src/variables.css

```diff
@@ -31,6 +32,9 @@
   --font-regular: HelsinkiGrotesk, var(--font-default);
   --font-medium: HelsinkiGrotesk-Medium, var(--font-default);
   --font-bold: HelsinkiGrotesk-Bold, var(--font-default);
+  --prose-width: 66ch;
```

Comment:
Is this sensible when using variable-width fonts?

Reply:
That's the one suggested by web.dev currently.
It handles scaling fonts with zoom (unlike px), and it handles different font-sizes (unlike rem). 66ch is around 650px with normal 18px latin fonts, but it scales better when the font is larger or smaller.
Max prose size is for legibility. 12px font is not legible at 650px, but 48px font is still legible at 1200px.

So it's the same as rem units but allows us to use 940px (container width) titles instead of clipping them to 650px as with rem units (assuming we use it for titles, currently it's only for text inputs).

For variable width fonts there might be an issue for example Helvetica 0 is thinner than 'm' so a line of only ms will overflow. But the only time this should happen is if we force no line-breaks, otherwise text should wrap automatically. The non contrived example I can think of is limiting a card title to two lines of 20ch and testing it with 0s instead of ms, but it does require forcing manual line-breaks.

Reply:
Ok, interesting. Can you do a demo on this next tue?